public/css/main.css

```diff
@@ -773,7 +773,7 @@ video#bgvid {
     background-color: white;
     width: 100%;
     height: 100%;
-    position: fixed;
+    position: relative;
```

Comment:
@sagec108 Could you rename the playback page's map's id? Both of these pages are sharing the same id which is creating this issue - with the current fix the playback page breaks. You can edit this branch.